variables.tf

```diff
@@ -26,6 +26,29 @@ variable "acl" {
   description = "The canned ACL to apply, defaults to `private`."
 }
 
+variable "use_acl" {
```

Comment:
Do we need this variable? If access_control_policy is not `null` could we then not use this instad of the canned ACL?

Reply:
I do see your point re the dynamic block and the if else statements, no objections there :-)